internal/hud/server/controller.go

```diff
@@ -31,6 +31,10 @@ func (s *HeadsUpServerController) Teardown(ctx context.Context) {
 }
 
 func (s *HeadsUpServerController) OnChange(ctx context.Context, st store.RStore) {
+	if s.hudServer.sailCli != nil {
+		s.hudServer.sailCli.MaybeBroadcast(st)
```

Comment:
it seems very weird for the hudServer to be responsible for this...can you elaborate a bit on what was wrong with making sailCli a Subscriber?

Reply:
...nothing? like, it felt a little weird and i wanted a gut check, and then I thought _you_ suggested putting the sail cli on the hud server? I can put it back.

Reply:
afair, you asked me how to trigger a Connect(), and I suggested calling Connect() from hudServer. I wouldn't expect that to imply moving all method calls here